controller_1.py

```diff
@@ -0,0 +1,14 @@
+from model import Model
+from view import View
+
+
+class Controller:
+    def __init__(self):
+        self.model = Model()
```

Comment:
as we discussed before, it is unnecessary to create an object of class Model because Model is a static class for database only. Therefore, it can create error

Reply:
However, if controller.py forced us to use it in this way, backend can fix their code to run. so don't worry